### ISSUE TITLE
README has out of date instructions for OSX

### ISSUE THREAD

**Issue by hamishdickson** [scala/scala-lang#356]
The README says users should run the command

```
gem install jekyll
``` 

OSX has gone rootless in El Capitan, so instead this should actually be (for OSX anyway)

```
sudo gem install -n /usr/local/bin/ jekyll
```

see http://stackoverflow.com/questions/31567029/how-can-i-install-jekyll-on-osx-10-11 for more details

I believe the part about updating ruby gems is still correct however

**Comment by SethTisue** [comment]
I've attempted to address this in https://github.com/scala/scala-lang/pull/361 by recommending that gems not be installed globally, but that Bundler be used instead.

**Comment by hamishdickson** [comment]
@SethTisue that sounds sensible, let me know if you need me to do anything

**Comment by soc** [comment]
"rootless" sounds a lot like "cloud". :-)

**Comment by SethTisue** [comment]
new attempt to address this: https://github.com/scala/scala-lang/pull/373

**Comment by SethTisue** [comment]
#373 was merged, see also #375. thanks for calling this to our attention.
